app/Console/Commands/CreateAdmin.php

```diff
@@ -22,48 +25,55 @@ class CreateAdmin extends Command
      */
     protected $description = 'Create a new admin user';
 
-    /**
-     * Create a new command instance.
-     *
-     * @return void
-     */
-    public function __construct()
-    {
-        parent::__construct();
-    }
-
     /**
      * Execute the console command.
      *
      */
     public function handle(): void
     {
-        // email address is taken as an argument
-        $email = $this->argument('email');
+        $name = text(
+            'Please input the name of the admin user',
+            'E.g. John Smith',
+            '',
+            true,
+            validate: ['name' => 'required|string|max:255']
+        );
 
-        // get the name as argument
-        $name = $this->argument('name');
+        $email = text(
+            'Please input the email of the admin user',
+            'E.g. user@example.com',
+            '',
+            true,
+            validate: ['email' => 'required|string|email:rfc,dns|max:255|unique:users']
+        );
 
-        // get the password as argument.
-        // Note, we have to take this as an argument, on our STRATO server the STDIN is not available
-        // therefore $this->ask and $this->secret won't work
-        $password = $this->argument('password');
+        $password = password(
+            'Please input the password for the admin user',
+            '',
+            true
+        );
 
-        $this->createUser($email, $name, $password);
+        progress(
+            'Creating admin user',
+            ['creating admin user'],
+            function () use ($name, $email, $password) {
+                $this->createUser($name, $email, $password);
+            }
+        );
     }
 
     /**
      * Create the user and assign admin rights
      *
-     * @param $email
-     * @param $name
-     * @param $password
+     * @param $email string
+     * @param $name string
+     * @param $password string
      */
-    private function createUser($email, $name, $password): void
+    private function createUser(string $name, string $email, string $password): void
     {
         $user = User::create([
-            'email' => $email,
             'name' => $name,
+            'email' => $email,
             'password' => Hash::make($password)
         ]);
         // Do stuff here to make the user an admin user
```

Comment:
If you want to mark the user as verified so that we don't have to go through manual email validation you can use the `markEmailAsVerified` method:

```suggestion
        // Do stuff here to make the user an admin user
        $user->markEmailAsVerified();
```